Client/src/CSS/ThemeWrapper.svelte

```diff
@@ -0,0 +1,11 @@
+<div>
+	<slot />
+</div>
+
+<style>
+	:global(:root) {
```

Comment:
ThemeWrapper is like a genral css file for the whole site, right?

Reply:
Yeah, I'm keeping it modular so that we (or the user) can change to color scheme easily

Reply:
Just wanted to make sure I understand